src/function/FunctionElem.ts

```diff
@@ -0,0 +1,71 @@
+import {graph} from "../graph/Graph";
+import {drawGraphElement, highlightCell, unHighlightCell} from "./FunctionDraw";
+import {getElementShape} from "./FunctionGetVars";
+import {paper} from "../main/DiagramCanvas";
+import {PackageNode} from "../datatypes/PackageNode";
+import {graphElement} from "../graph/GraphElement";
+import {addClass, addVocabularyElement, createNewElemIRI} from "./FunctionCreateVars";
+import {parsePrefix} from "./FunctionEditVars";
+import {ProjectElements, ProjectSettings, VocabularyElements} from "../config/Variables";
+
+export function resizeElem(id: string) {
+	let view = paper.findViewByModel(id);
+	if (view) {
+		let bbox = view.getBBox();
+		let cell = graph.getCell(id);
+		let links = graph.getConnectedLinks(cell);
+		for (let link of links) {
```

Comment:
consider parameterizing these loops with the respective function - link.source/link.target and reuse the code - the code is quite repeated.